index.js

```diff
@@ -500,45 +517,24 @@ RingPop.prototype.lookupN = function lookupN(key, n) {
 };
 
 RingPop.prototype.reload = function reload(file, callback) {
```

Comment:
It's not used in ringpop-admin and (according to our stats backend) never called in production.